snapcraft/plugins/v2/cmake.py

```diff
@@ -48,27 +48,43 @@ def get_schema(cls) -> Dict[str, Any]:
                     "uniqueItems": True,
                     "items": {"type": "string"},
                     "default": [],
+                },
+                "cmake-generator": {
+                    "type": "string",
+                    "default": "Unix Makefiles",
                 }
             },
         }
 
+
+    def _is_ninja(self) -> bool:
+        return "Ninja" in self.options.cmake_generator
```

Comment:
Would compare directly, though I'm curious if there are (plans for) more than 1 ninja cmake generator option?